data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			`	You're greeted by an Arach, who came in what you believe to be their version of a car, though you gave up on trying to guess the exact purpose of each pipe and plating, why some wheels weren't touching the ground, while pondering if you would comfortably fit inside one.`
+			`	The Arach takes a translation device out of his pocket, and formally greets you, "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`
+			choice
+				`	"Are you the shipwright I'll be taking to Lugglop?"`
+					goto servant
+				`	"What's with this place? Why are there so many buildings all over?"`
+					goto property
+			label servant
+			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as chauffeur, I work."`
+				goto mistress
+			label property
+			`	"No mere mansion, the private property of House Mallob is," he says, as he starts pointing to different areas. "A museum, there we have, detailing the history of Arachi ships. To the left, a shopping center is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
+			label mistress
+			`	He asks that you go inside the vehicle, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must."`
+			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`
+			`	The entrance to the mansion isn't as castle-like as the walls, though it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`
+			`	He guides you through the mansion, as you pass through several other servants, some rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students on the way, here to study with the shipwrights of House Mallob.`
+			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
+			choice
+				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`
+				`	"A Kimek? I thought this was an Arach House."`
+			`	The butler ignores what you said, and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student on the university here, she was."`
+			`	Finally, the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`
```

Comment:
```suggestion
			`	Finally, the reason why you had to come all the way to this part of <origin> becomes clear; Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she keeps asking more and more questions ranging from how you acquired a Jump Drive to all the places you've been.`
```
Everything here is technically optional (personal preference), but I would highly recommend either these changes be made or this paragraph rewritten.